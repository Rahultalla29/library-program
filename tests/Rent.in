RENT 100000 111111
ADD MEMBER Jeff
RENT 100000 111111
ADD COLLECTION sample1.csv
RENT 100000 111999
RENT 100010 111111
RENT 100000 111111
RENT 100000 111112
RENT 100000 111113
RENT 100000 111113
RENT 100010 111111
RELINQUISH ALL 100000
MEMBER HISTORY 100000
EXIT
