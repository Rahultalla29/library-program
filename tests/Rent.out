user: No members in system.

user: Success.

user: No books in system.

user: 17 books successfully added.

user: No such book in system.

user: No such member in system.

user: Success.

user: Success.

user: Success.

user: Book is currently unavailable.

user: No such member in system.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)

user: Ending Library process.
